src/m365/spo/commands/homesite/homesite-remove.ts

```diff
@@ -74,7 +74,7 @@ class SpoHomeSiteRemoveCommand extends SpoCommand {
         }
       }
       catch (err: any) {
-        this.handleRejectedPromise(err);
+        this.handleRejectedODataJsonPromise(err);
```

Comment:
Are you sure this endpoint returns an odata / json response? Otherwise we'd better use the original one.

Reply:
It can throw an odata/json response, as we are calling the `/_api/contextinfo` when calling `spo.getRequestDigest()`.

I know that it will probably never fail, but you never know ;-)